src/components/submission/filters/review-state.vue

```diff
@@ -45,9 +45,18 @@ const options = reviewStates.map(reviewState => ({
 }));
 
 const { t } = useI18n();
-const placeholder = (counts) => t('placeholder', counts);
+const placeholder = (counts) => {
+  if (counts.total === counts.selected) return t('noReviewStateSelected');
```

Comment:
common.none

as i flag these (none)s, i think maybe matt would point out that different (none)s for different types of filters would be referenced differently? but probably not?